process/jira.md

```diff
@@ -0,0 +1,54 @@
+# Jira Hygiene
+
+As almost anything even Jira requires regular maintenance to address:
+
+-   Incomplete/unclear tickets
+-   Stale/outdated tickets
+-   Forgotten/unpopular tickets
+-   Usefulness of Jira content as a whole
+
+Why? Even though it might not look like it at first glance, Jira is a tool to *communicate with* people in an open, transparent and remote-friendly manner and facilitate *reaching our goals*. This text tries to apply our [PLOW](https://culture.mayadata.io/plow/plow.html) to working with Jira. To keep various Jira projects open and welcoming. To allow us to cooperate effectively and win - personally and professionally and deliver the value that is useful to the company and open-source community in a repeatable and dependable manner.
```

Comment:
```suggestion
Why? Even though it might not look like it at first glance, Jira is a tool to *communicate with* people in an open, transparent and remote-friendly manner and facilitate *reaching our goals*. This article tries to apply our [PLOW](https://culture.mayadata.io/plow/plow.html) to working with Jira. To keep various Jira projects open and welcoming. To allow us to cooperate effectively and win - personally and professionally and deliver the value that is useful to the company and open-source community in a repeatable and dependable manner.
```